pkg/consensus/consensus.go

```diff
@@ -286,6 +286,7 @@ events:
 		select {
 		case <-s.quit:
 			s.dbft.Timer.Stop()
+			s.Chain.UnsubscribeFromBlocks(s.blockEvents)
```

Comment:
It'd be nice to have this in a separate PR to be merged into the `master` branch.

Reply:
https://github.com/nspcc-dev/neo-go/pull/2566.